src/routes/transfers.js

```diff
@@ -18,6 +18,11 @@ router.post(
   transfersController.updateTransferSteps,
 );
 
+router.post(
+  '/update/edges',
+  validate(transfersValidation.findTransferSteps),
```

Comment:
you are using this validation function, but these are not the correct parameters, right¿?